PurchasesCoreSwift/Logging/Strings/Emoji.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Emoji.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/8/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+enum Emoji: String {
+    case apple = "\u{1F34E}\u{FE0F}" // apple emoji
+    case doubleExclamation = "\u{203C}\u{FE0F}" // red double exclamation mark emoji
+    case heartCatEyes = "\u{1F63B}\u{FE0F}" // heart cat eyes emoji
+    case info = "\u{2139}\u{FE0F}" // info emoji
+    case moneyBag = "\u{1F4B0}\u{FE0F}" // money bag emoji
+    case sadCatEyes = "\u{1F63F}\u{FE0F}" // sad cat eyes emoji
+    case person = "\u{1F464}\u{FE0F}" // user emoji
+    case warning = "\u{26A0}\u{FE0F}"  // yellow warning emoji
```

Comment:
I think the comments aren't needed here anymore, the emoji names seem descriptive enough